doc/EXAMPLES.md

```diff
@@ -1094,3 +1094,33 @@ public class DemoRule extends JRule {
   }
 }
 ```
+
+## Example 43 - Creating a rule dynamically using JRuleBuilder
```

Comment:
missed in the examples overview at the top

Reply:
Added missing link in the overview